pkg/cluster/describe.go

```diff
@@ -217,9 +218,9 @@ func PrintClusterDescription(connection *sdk.Connection, cluster *cmv1.Cluster)
 		apiURL,
 		apiListening,
 		cluster.Console().URL(),
-		cluster.Nodes().Master(),
-		cluster.Nodes().Infra(),
-		computesStr,
+		printNodeInfo(fmt.Sprintf("%d", cluster.Nodes().Master()), cluster.AWS().AdditionalControlPlaneSecurityGroupIds()),
+		printNodeInfo(fmt.Sprintf("%d", cluster.Nodes().Infra()), cluster.AWS().AdditionalInfraSecurityGroupIds()),
```

Comment:
Why not the use of `strconv.Itoa()` instead of the string formatting

Reply:
Mostly because this was the pattern prior, adjusted for compute replicas too